css/src/components/form/form.scss

```diff
@@ -52,7 +52,8 @@ $field-error-color: $danger !default;
 		}
 	}
 
-	.field-error {
+	.field-error,
+	.field-error-custom {
```

Comment:
just for my edification, if `.field-error` and `.field-error-custom`  are the same thing, why do they both exist? Why not use only one?

Reply:
The basic validation in form behavior looks for the first `.field-error` class and updates its text content with a message when there's an error, or empties it if validation passes. That's ok when there's one input within a field as there can only be one message. 

For markdown editor, there's both a text area and an input so there's a potential of two messages, one for the text area and one for the input. The input requires custom validation outside of the basic validation provided by form behavior. The custom validation is run once `form-validating` event is dispatched, its message is applied to `.field-error`. Then, the basic validation runs for the text area, finds the same `.field-error` that already has a message, and overwrites it with the message for the text area. Using a different class for the custom validation error message would avoid this scenario.

Reply:
It's possible for the custom validation error message to be something like this instead: 

`<p class="field-error">Textarea must have at least 10 characters."</p>`
`<p class="color-danger font-size-sm line-height-something">Maximum number of attachments exceeded. Please select up to 1 attachments.</p>`

vs.

`<p class="field-error">Textarea must have at least 10 characters."</p>`
`<p class="field-error-custom">Maximum number of attachments exceeded. Please select up to 1 attachments.</p>`

Reply:
Chatted with Will, decided to completely decouple `.field-error` from the form behavior functionality, use data attributes instead. Reverted `.field-error-custom`